enterprise/backend/src/metabase_enterprise/advanced_config/file/api_keys.clj

```diff
@@ -50,37 +55,50 @@
 
 (defn- init-from-config-file!
   [api-key-config]
-  (let [{:keys [name key group creator]} api-key-config
-        group-id (case group
-                   "admin" (u/the-id (perms/admin-group))
-                   "all-users" (u/the-id (perms/all-users-group)))
-        unhashed-key (u.secret/secret key)
-        creator (get-admin-user-by-email creator)]
+  (let [{:keys [name key group creator]} api-key-config]
+    ;; Validate key format
+    (when-not (and (string? key)
```

Comment:
Is this validation necessary? Isn't the spec above enforced already? If not maybe we can enforce it manually here (but use it, so we aren't defining the same requirements twice)